tests/e2e/cucumber/features/smoke/spaces/publicLink.feature

```diff
@@ -87,9 +87,15 @@ Feature: spaces public link
       """
       Unfortunately, your password is commonly used. please pick a harder-to-guess password for your safety
       """
+    And "Alice" closes the public link password dialog box
     When "Alice" tries to sets the password of the public link named "myPublicLink" of resource "lorem.txt" to "12345678"
     Then "Alice" should see an error message
       """
       Unfortunately, your password is commonly used. please pick a harder-to-guess password for your safety
       """
+    And "Alice" shows the password of the public link
```

Comment:
Can we use the word `reveal` here? Ik they mean similar things but reveal would be more like it was hidden in the first place which I think makes sense here
```suggestion
    And "Alice" reveals the password of the public link
```